tests/features/conftest.py

```diff
@@ -290,6 +290,7 @@ def value(self) -> int:
     Key('^_', b'^_', '\x1f'),
     Key('^\\', b'^\\', '\x1c'),
     Key('!resize', b'KEY_RESIZE', curses.KEY_RESIZE),
+    Key('^D', b'KEY_DC', curses.KEY_DC),
```

Comment:
I think this should map `^D` to `^D` with the proper code here, that way we're actually testing that the translation occurs

I believe the key for ^D is `\x04` -- I determined this using `babi --key-debug`

Reply:
Yes,  here is how babi reports usage of it when I tried it before these changes:
![Screenshot from 2020-10-26 13-46-41](https://user-images.githubusercontent.com/5195944/97392542-b00d2f00-18ea-11eb-94eb-82702fe5ca4f.png)

Using \x04 in place of curses.KEY_DC passes also all the tests. What do you think of the most recent commit I just added to change the above?